app/src/main/java/it/quip/android/view/RoundedImageTransform.java

```diff
@@ -0,0 +1,45 @@
+package it.quip.android.view;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.graphics.Shader;
+
+public class RoundedImageTransform implements com.squareup.picasso.Transformation {
+
```

Comment:
As opposed to the separate library for rounded image views? Yeah I like this solution better. We have more control over the transform and we don't have to bring in another library just for views